tests/integ/test_workflow_with_clarify.py

```diff
@@ -0,0 +1,288 @@
+    analysis_config["methods"]["report"] = {"name": "report", "title": "Analysis Report"}
+
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        analysis_config_file = os.path.join(tmpdirname, "analysis_config.json")
+        with open(analysis_config_file, "w") as f:
+            json.dump(analysis_config, f)
+        config_input = ProcessingInput(
+            input_name="analysis_config",
+            source=analysis_config_file,
+            destination="/opt/ml/processing/input/config",
+            s3_data_type="S3Prefix",
+            s3_input_mode="File",
+            s3_compression_type="None",
+        )
+
+        data_input = ProcessingInput(
+            input_name="dataset",
+            source=data_config.s3_data_input_path,
+            destination="/opt/ml/processing/input/data",
+            s3_data_type="S3Prefix",
+            s3_input_mode="File",
+            s3_data_distribution_type=data_config.s3_data_distribution_type,
+            s3_compression_type=data_config.s3_compression_type,
+        )
+
+        result_output = ProcessingOutput(
+            source="/opt/ml/processing/output",
+            destination=data_config.s3_output_path,
+            output_name="analysis_result",
+            s3_upload_mode="EndOfJob",
+        )
+
+        processor = SageMakerClarifyProcessor(
+            role="SageMakerRole",
+            instance_count=instance_count,
+            instance_type=instance_type,
+            sagemaker_session=sagemaker_session,
+        )
+
+        property_file = PropertyFile(
+            name="BiasOutput",
+            output_name="analysis_result",
+            path="analysis.json",
+        )
+
+        step_process = ProcessingStep(
+            name="my-process",
+            processor=processor,
+            inputs=[data_input, config_input],
+            outputs=[result_output],
+            property_files=[property_file],
+        )
+
+        cond_left = JsonGet(
+            step=step_process,
+            property_file="BiasOutput",
+            json_path="post_training_bias_metrics.facets.F1[0].metrics[0].value",
+        )
+
+        step_condition = ConditionStep(
+            name="bias-condition",
+            conditions=[ConditionLessThanOrEqualTo(left=cond_left, right=1)],
+            if_steps=[],
+            else_steps=[],
+        )
+
+        pipeline = Pipeline(
+            name=pipeline_name,
+            parameters=[instance_type, instance_count],
+            steps=[step_process, step_condition],
+            sagemaker_session=sagemaker_session,
+        )
+
+        try:
+            response = pipeline.create(role)
+            create_arn = response["PipelineArn"]
+
+            execution = pipeline.start(parameters={})
+
+            response = execution.describe()
+            assert response["PipelineArn"] == create_arn
+
+            try:
+                execution.wait(delay=30, max_attempts=60)
```

Comment:
30 minute test. any way we can time out sooner?

Reply:
The test actually runs for 20-25 mins, so added a 30 min delay. It does a lot of things from running training job, create model and deploy model and then run processing job to calculate bias.
Only way to reduce the test time is to just calculate pre-training bias instead of post training bias. For that we won't need to train and deploy the model. Will just need to run processing job.
But with post-training-bias, we can verify the entire flow. Let me know if you want me to update